domains/auth/Auth.API/API/Services/OidcProviders/SignaturGruppen.cs

```diff
@@ -34,4 +36,24 @@ public NextStep CreateAuthorizationUri(AuthState state)
 
         return authorizationUri;
     }
+
+    public async Task Logout(string token)
+    {
+        var url = Configuration.GetOidcUrl();
+        httpClient.BaseAddress = new Uri(url);
+
+        var response = await httpClient.PostAsJsonAsync(
+            "/api/v1/session/logout",
+            new { id_token = token }
+            );
```

Comment:
```suggestion
        var response = await httpClient.PostAsJsonAsync("/api/v1/session/logout", new { id_token = token });
```

Reply:
Do we have a linter so we format code the same way?